pkgs/applications/networking/cluster/lens/default.nix

```diff
@@ -24,6 +24,9 @@ appimageTools.wrapType2 {
     ''
       mv $out/bin/${name} $out/bin/${pname}
 
+      wrapProgram $out/bin/${pname} \
```

Comment:
So this is an interesting problem. You don't have `wrapProgram` because you don't have `makeWrapper` in your `nativeBuildInputs`. But it's an appimage builder so you don't control the inputs

It looks like losslesscut-bin does a `source "${makeWrapper}/nix-support/setup-hook"` right before their `wrapProgram` call and makes that an argument to the package file to get around that. That seems quite ugly, but I don't have a better suggestion really so I guess do that if you are interested in wayland support

Reply:
Alright, I've fixed that.